src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -146,7 +146,8 @@
 
     // field-specific error messages
     public static final String HINT_FOR_CORRECT_EMAIL =
-            "An email address contains some text followed by one '@' sign followed by some more text. "
+            "An email address contains some text followed by one '@' sign followed by some more text, and should end "
+                    + "with a top domain address like .com. "
```

Comment:
"top domain" has no meaning. Top-level domain (TLD) is the universally understood term for what is mentioned here.